src/components/extensionTopContent/top-contents/GlobalNavigationBar.jsx

```diff
@@ -4,25 +4,39 @@ import { useContext } from "react";
 
 import ExtensionContext from "../../../context/ExtensionContext";
 
-function GnbButton({ iconShapeType }) {
+function GnbButton({ iconShapeType, onClick }) {
   return (
     <button className="w-8 text-lg">
-      <FontAwesomeIcon icon={iconShapeType} />
+      <FontAwesomeIcon
+        icon={iconShapeType}
+        onClick={onClick}
+      />
     </button>
   );
 }
 
 export default function GlobalNavigationBar() {
   const { bookmarkList } = useContext(ExtensionContext);
 
+  const hendleOnClickOpenOptionPage = () => {
```

Comment:
```suggestion
  const handleOnClickOpenOptionPage = () => {
```
함수명에 오타가 포함된 것 같습니다. 수정하면 좋을 것 같습니다. 감사합니다!

Reply:
좋은 검토 감사합니다. 덕분에 실수를 확인 할 수 있었습니다. 참조하여 반영하겠습니다.